src/api/revurderingApi.ts

```diff
@@ -42,6 +43,92 @@ export async function opprettRevurdering(
     });
 }
 
+export async function opprettStans(
+    sakId: string,
+    fraOgMed: Date,
+    årsak: OpprettetRevurderingGrunn,
+    begrunnelse: string
+): Promise<ApiClientResult<StansAvYtelse>> {
+    return apiClient({
+        url: `/saker/${sakId}/revurderinger/stans`,
+        method: 'POST',
+        body: {
+            fraOgMed: formatISO(fraOgMed, { representation: 'date' }),
+            årsak: årsak,
+            begrunnelse: begrunnelse,
+        },
+    });
+}
+
+export async function oppdaterStans(
+    sakId: string,
+    revurderingId: string,
+    fraOgMed: Date,
+    årsak: OpprettetRevurderingGrunn,
+    begrunnelse: string
+): Promise<ApiClientResult<StansAvYtelse>> {
+    return apiClient({
+        url: `/saker/${sakId}/revurderinger/stans/${revurderingId}`,
+        method: 'PATCH',
+        body: {
+            fraOgMed: formatISO(fraOgMed, { representation: 'date' }),
+            årsak: årsak,
+            begrunnelse: begrunnelse,
+        },
+    });
+}
+
+export async function gjenoppta(args: {
```

Comment:
Hvorfor bruker vi `args`-objektet noen plasser og andre ikke?

Reply:
Har wrappat in den i ett objekt sån att vi kan bruke våre hooks (`useApiCall` & `useAsyncActionCreator`) som gör det mycket enklare. (De funksjonerne støtter ikke når det er flere parametrer)

Reply:
Det er greit det. Men jeg lurte på hvorfor noen av de nye funksjonene brukte args og noen ikke. Tenkte vi kunne prøve kjøre samme kodestil. I hvert fall i samme fil?

Reply:
Hehe borde vara samme, tror ikke jag gjort det når de ikke brukes i de hooksen. Men burde uansett vara lik stil! Kan fikse så de andra nye funksjonerna også er wrappade i ett objekt. 😊